commerce_coordinator/apps/commercetools/sub_messages/tasks.py

```diff
@@ -72,7 +72,7 @@ def fulfill_order_placed_message_signal_task(
 
     default_params = {
         'email_opt_in': True,  # ?? Where?
-        'order_number': order.id,
+        'order_id': order.id,
```

Comment:
We need both, can you please add the other one? In the future [ticket](https://2u-internal.atlassian.net/browse/SONIC-330), order_number value is going to be updated

```suggestion
        'order_number': order.id,
        'order_id': order.id,
```